### DISCUSSION TITLE
Sync Error / Project Loss

### DISCUSSION THREAD

**Discussion by mauricioranzan** [opengisch/QField#3589] [Q&A]
Hello.
We carried out a forest survey of 171 trees, where I record data from each one and attach 2 photos.
I've done several projects and it always works.
This time, today we went to sync the field project from the smartphone to the cloud and what happened was that we lost the entire project.
The project now appears empty.
I'm desperate.
Does anyone know if there is any way to get it back?

**Comment by suricactus** [comment]
Hello @mauricioranzan . Can you please share you username and project name on QFieldCloud? Also exact steps you followed and what it means "appears empty"? All layers were deleted? Or the collected features are unavailable?

**Reply by mauricioranzan** [reply]
Hi Ivan Ivanov.
How are you?

My username is "environmental".
The name of my project in QField Cloud is "Laurindo_Sachet_BR_SIG_AUC_SUPRESSAO_qfield".

The steps were:
1. I have a blank template in QGis on PC;
2. Export it to a QField Cloud project via the plugin and save it in a client folder;
3. I open QFieldCloud on my smartphone and download the project;
4. I carry out the field survey;
5. Return to the office and sync through the APP;
6. I open the project saved in the client's folder and synchronize using the plugin, choosing the option to prefer Cloud;

I've done this several times and it always works. Sometimes it takes time to download the project on the smartphone, but it works.

This time all data disappeared.
The layers remained, but the resources collected in the field disappeared, with the exception of the photos that were saved in a smartphone folder.

**Reply by suricactus** [reply]
Thanks for the info, @mauricioranzan . Your changes were successfully recorded in QFieldCloud, but unsucessfully applied to your project for unknown reason. You can check them in the web interface "Project -> Laurindo_Sachet_BR_SIG_AUC_SUPRESSAO_qfield -> Changes" page. There you can try to reapply your changes and hopefully this time they will work.

**Reply by mauricioranzan** [reply]
Can I do this via the APP or PC?

**Reply by suricactus** [reply]
Via web browser, preferrably on desktop, but not required.
